### DISCUSSION TITLE
How to set up MipMaps for a loaded texture?

### DISCUSSION THREAD

**Discussion by Chewico3d** [stride3d/stride#1647] [Q&A]
I'm just making a game where I need to generate a mipmaps from a loaded texture
` Texture.Load(EnviromentPack.graphics,File.OpenRead(OutputPath), TextureFlags.RenderTarget,GraphicsResourceUsage.Immutable,true);`

The problem is that I can't find a way to generate mipmaps.

![image](https://user-images.githubusercontent.com/62964377/231546199-6fff9dd1-2f2f-47d2-a323-e0080e6284d0.png)

With the textures loaded in the inspector you can generate them but you can't with loaded ones.

This is what the game looks like without mipmaps
![image](https://user-images.githubusercontent.com/62964377/231546487-0eada662-9127-4c9a-b6b9-9c5bf4b6d555.png)

(Yes, it's a minecraft clone)

**Comment by tebjan** [comment]
The mipmap operations are in the Image/Texture package/project in the source code, it theoretically belongs to the editor side and should not be used at runtime. But if you drill down, you might find a few methods to load texture with mips.
